unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1437,6 +1437,12 @@ handleFindI isVerbose fscope ws input = do
       -- results may contain things in lib.
       pped <- Cli.currentPrettyPrintEnvDecl
       pure (pped, names)
+    FindLocalSubnamespace p -> do
+      path <- Cli.resolvePath p
+      branch0 <- Cli.getBranch0At path
+      let names = Branch.toNames branch0
+      pped <- Cli.currentPrettyPrintEnvDecl
+      pure (pped, names)
```

Comment:
I think we should still filter out libs from these names with `Branch.withoutLib`, e.g. if you're in loose-code and do `.> find-in myprojects.base.releases.M2` or w/e